README.md

```diff
@@ -102,12 +102,13 @@ For front-end applications and node <v4, please use `require('fluture/es5')`.
 [<img src="https://raw.github.com/fantasyland/fantasy-land/master/logo.png" align="right" width="82" height="82" alt="Fantasy Land" />][FL]
 [<img src="https://raw.githubusercontent.com/rpominov/static-land/master/logo/logo.png" align="right" height="82" alt="Static Land" />][6]
 
-Fluture implements [FantasyLand 1][FL1], [FantasyLand 2][FL2],
-[FantasyLand 3][FL3] and [Static Land][6] compatible `Functor`, `Bifunctor`,
-`Apply`, `Applicative`, `Chain`, `ChainRec` and `Monad`.
-
-FantasyLand 0.x is *mostly* supported. Everything but `Apply` (`ap`) is, this
-means that dispatchers to FantasyLand 0.x `ap` will not work.
+* `Future` implements [FantasyLand 1][FL1], [FantasyLand 2][FL2],
+  [FantasyLand 3][FL3] and [Static Land][6] compatible `Bifunctor`, `Monad` and
```

Comment:
Suggestions:

  - consider a comma after "FantasyLand 3";
  - consider a hyphen before "compatible" (i.e. _and Static Land -compatible_); and
  - consider writing Fantasy Land as two words (as it is written in the [spec][1]).


[1]: https://github.com/fantasyland/fantasy-land